src/data/location.json

```diff
@@ -647,6 +663,7 @@
     "backMarker": {
       "to": 18,
       "position": { "x": 580, "y": -574, "z": -4925 },
+	  "lookAt": { "x": -837.31, "y": 241.71, "z": 4913.17 },
```

Comment:
This should probably look towards the art gallery since it's directly behind this nav spot

Reply:
I'm happy with both changes! :)